test/fixtures/permission/fs-read.js

```diff
@@ -21,6 +21,11 @@ const regularFile = __filename;
     permission: 'FileSystemRead',
     resource: path.toNamespacedPath(blockedFile),
   }));
+  fs.readFile(bufferBlockedFile, common.expectsError({
```

Comment:
non-blocking nit... this particular test is rather dense. While we're in here making changes it would be nice to fixup the formatting just a bit so that it's easier to read and see what is going on. Comments explaining what is being tested would be helpful also.